### PR TITLE
Fortunac/plugin tests

### PR DESCRIPTION
**Pull request by fortunac**
Refers to #123.

Big change to `test_wp.ml`. Instead of building a command in the test file, we run the scripts from the `sample_binaries` directly. That way we can actually test our plugin's default flags and test that our scripts work. There used to be a mismatch of default flags when we were building the commands in the test file.

### CODE REVIEW

File: wp/resources/sample_binaries/retrowrite_stub/run_wp_inline_afl.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Stubs the lines of asssembly that retrowrite addsto the beginning of each
```
Comment (codyroux) [inline wp/resources/sample_binaries/retrowrite_stub/run_wp_inline_afl.sh]: did you run spell-check on these?

File: wp/resources/sample_binaries/retrowrite_stub_no_ret/run_wp.sh [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# Stubs the lines of asssembly that retrowrite adds to the beginning of each
```
Comment (codyroux) [inline wp/resources/sample_binaries/retrowrite_stub_no_ret/run_wp.sh]: Typo

**Comment by codyroux** [comment]
I'm very impressed, this is good work and should result in a large quality of life improvement.

File: wp/resources/sample_binaries/retrowrite_stub/run_wp_inline_afl.sh (hunk 1 above)
Comment (fortunac) [reply wp/resources/sample_binaries/retrowrite_stub/run_wp_inline_afl.sh]: Apparently not. :)

**Pull request merged**